data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -496,10 +496,11 @@ def _prepare_initial_df(self) -> pd.DataFrame:
         # >= some threshold.
         # TODO: Add more fields here.
         #  https://github.com/usds/justice40-tool/issues/970
-        ReversePercentile = namedtuple(
-            typename="ReversePercentile",
-            field_names=["field_name", "low_field_name"],
-        )
+        @dataclass
```

Comment:
I did this so the newer version of pylint can parse this file on python 3.8. I know @esfoobar-usds reverted pylint, but I'm leaving it so when we do upgrade, everything is copacetic.

Reply:
Thank you! I will try another time with a branch off of Narwahl to see if the pylint errors resolved after you merge this